tests/test_toa_reader.py

```diff
@@ -52,3 +77,27 @@ def test_jump_3(self):
 
     def test_obs(self):
         assert self.x.table[1]["obs"] == "gbt"
+
+    def test_ephem(self):
+        assert self.x.ephem == "DE421"
+
+    def test_planets(self):
+        assert self.x.planets is False
+
+    def test_clock(self):
+        assert self.x.clock_corr_info["bipm_version"] == bipm_default
+
+    def test_model_override1(self):
```

Comment:
What does the "override" here mean? It would be good to have tests to check that values specified as keyword arguments to `get_TOAs` override values obtained from the model, but that's not what this checks.

Reply:
The model is overriding the default behavior of the TOA reading code.  But I can also check the overriding of the override!